### PR TITLE
Example files

### LINKED ISSUE #174: Ditto reader doesn't work in a virtualenv
### Expected behavior

Ditto reader should work when installed in a virtualenv

### Actual behavior

See comments in #120

### LINKED ISSUE #192: Add electrical components to feature-file to work with opendss
**Is your feature request related to a problem? Please describe.**
[Feature-file](https://github.com/urbanopt/urbanopt-cli/blob/develop/example_files/example_project.json) does not include any electrical equipment, which is required for OpenDSS to operate.

**Describe the solution you'd like**
Bring in electrical config from [ditto-reader geojson file](https://github.com/urbanopt/urbanopt-ditto-reader/blob/develop/example/urbanopt_example.json).

### PR DESCRIPTION
**Pull request by vtnate**
### Pull Request Description

Uses an installation of [ditto-reader](https://pypi.org/project/urbanopt-ditto-reader/) (which includes the cli for that repo) to make the `uo opendss` command work as expected. I'll repeat for the folks in the back: **ditto reader must be installed beforehand!** During testing, you must install from the `cli` branch of ditto-reader. If you install ditto-reader into a virtualenv (good practice), ensure that virtualenv is activated when you call `uo opendss`.

Also includes rspec tests, which for dev work should be excluded (change the test from `it 'does a thing' do...` to `xit 'does a thing' do...` for the test to run the electrical_network and to run opendss (lines 224 & 268, respectively). The only reason to exclude these tests is because it takes a long time to run a scenario of 13 large buildings. Less dev hairpulling if you skip these two tests.

### Checklist

I expect the `opendss` test to fail, because Jenkins does not have Python & ditto-reader installed, which that test depends on.

- [x] Unit tests have been added or updated
- [ ] All ci tests pass (green)
- [x] An [issue](https://github.com/urbanopt/urbanopt-cli/issues) has been created (which will be used for the changelog)
- [x] This branch is up-to-date with develop

### CODE REVIEW

File: lib/uo_cli.rb [hunk 1]
```diff
@@ -653,36 +654,8 @@ def self.check_reader
...
-        r = UrbanoptDittoReader.new(pconf)
-        r.run
-      rescue StandardError => e
-        abort("\nOpenDSS run did not complete successfully: #{e.message}")
-      end
-
-      puts "\nDone. Results located in #{config['opendss_folder']}\n"
-
+      # We're calling the python cli that gets installed when the user installs ditto-reader. Haaaaaaaacky!
+      system("uo_cli run-opendss -s #{@opthash.subopts[:scenario]} -f #{@opthash.subopts[:feature]}")
```
Comment (kflemin) [inline lib/uo_cli.rb:658]: we need to rename the other CLI...it's going to be too confusing.  how about ditto-reader-cli or uo-ditto-cli?  something that ties it back to that repo would be great!

**Review by kflemin — APPROVED** [review]
🎉🎉🎉🎉🎉🎉🎉. yay so exciting! I'm sure this will be much more reliable than the pycall gem.  Can you get @tanushree04 or @rawadelkontar to test on windows...that's where the painpoint always is!

**Comment by kflemin** [comment]
@vtnate let's make sure not to merge until we have a release of urbanopt-ditto-reader up on pypi!

**Comment by vtnate** [comment]
Test failure is expected because Jenkins is not installing ditto-reader which is required for the opendss test to pass.

**Comment by vtnate** [comment]
@kflemin, @tanushree04 reviewed and I made a small addition on her suggestion. Confirmed it works on Windows!

**Comment by vtnate** [comment]
`Do not merge` flag has been removed since urbanopt-ditto-reader is now on pypi

File: lib/uo_cli.rb [hunk 2]
```diff
@@ -655,13 +667,29 @@ def self.check_reader
...
       # We're calling the python cli that gets installed when the user installs ditto-reader.
-      # If ditto-reader is installed into a venv (recommended), that venv must be activated for this to work.
-      # Haaaaaaaacky!
-      begin
-        system("ditto_reader_cli run-opendss -s #{@opthash.subopts[:scenario]} -f #{@opthash.subopts[:feature]}")
-      rescue FileNotFoundError
-        abort("\nMust post-process results before running opendss. We recommend --default." \
-        "Once opendss is run, you may then 'process --opendss'")
+      # If ditto-reader is installed into a venv (recommended), that venv must be activated when this command is called.
+      if @opthash.subopts[:scenario] && @opthash.subopts[:feature]
```
Comment (vtnate) [inline lib/uo_cli.rb:671]: @kflemin I can't pass a flag to the python cli if it hasn't been mentioned in the ruby cli. I could make a big nest with all options individually listed, but that feels icky. Do you have a recommendation for a better way?

**Review by kflemin — APPROVED** [review]

**Pull request merged**